rust/src/storage/file/mod.rs

```diff
@@ -148,7 +148,8 @@ mod tests {
             .put_obj(new_file_path.to_str().unwrap(), b"hello")
             .await
         {
-            panic!(format!("Expect put_obj to return Ok, got Err: {:#?}", e));
+            let error_message = format!("Expect put_obj to return Ok, got Err: {:#?}", e);
+            panic!("{}", error_message);
```

Comment:
would be better to rewrite it as `panic!("Expect put_obj to return Ok, got Err: {:#?}", e)`

Reply:
👍